docs/content/v0.20/README.md

```diff
@@ -49,7 +50,7 @@ including OCM Klusterlet for the WECs.
 ## Setup
 
 1. Install [KubeFlex](https://github.com/kubestellar/kubeflex/tree/main#installation). 
-Make sure you have at least version 0.3.3 or higher. To upgrade from an existing installation, 
+Make sure you have at least version v0.4.0 or higher. To upgrade from an existing installation, 
```

Comment:
Need to update the pre-req listing here: https://github.com/kubestellar/kubestellar/blob/ks-0.20/test/pre-reqs.md

Reply:
done